packages/sanity/src/structure/panes/documentList/sheetList/SheetListCell.tsx

```diff
@@ -157,7 +157,9 @@ export function SheetListCellInner(props: SheetListCellInnerProps) {
     'aria-selected': !!cellState,
     'data-testid': cellId,
     'id': cellId,
-    'ref': (ref: InputRef) => (inputRef.current = ref),
+    'ref': (ref: InputRef) => {
+      inputRef.current = ref
+    },
```

Comment:
This were only caught during runtime thanks to StrictMode, the linter or TSC did not since we're spreading props instead of using JSX:
```
<Select
  ref={(ref: InputRef) => (inputRef.current = ref)}
/>
```

[In React 19 ref callbacks can return cleanup functions the same way `useEffect` can](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#ref-cleanup-required), and thus implicit returns in ref callbacks is no longer allowed.

Reply:
> oh, but I see we have the `no-void` eslint rule enabled. oh well

no fun allowed 😩